src/js/pouch-express.js

```diff
@@ -207,28 +210,19 @@ gpii.pouch.express.cleanup = function (that) {
             cleanupPromises.push(function () { return databaseInstance.destroyPouch(); });
         });
 
-        var logCleanupPromise = gpii.pouchdb.timelyRimraf(that.options.expressPouchLogPath, {}, that.options.rimrafTimeout);
-        cleanupPromises.push(logCleanupPromise);
+        cleanupPromises.push(function () {
+            kettle.test.deleteFolderRecursive(that.options.expressPouchLogPath);
```

Comment:
As we mentioned in private chat, if we do mean to use this it should be hoisted out of the kettle.test implementation. However, I can't see this or any utility like it making a substantial difference to our cleanup story on Windows.